### PR TITLE
Delay shutting down the PC

### PR DESCRIPTION
**Pull request by stefanroelofs**
Add a check if delay is empty.

### CODE REVIEW

**Comment by pvizeli** [comment]
please update version

**Comment by stefanroelofs** [comment]
I updated the addon to use bashio. 
It seems that the checks are failing, but I don't know why..
I'm not too sure about the function call in run.sh line 19. Do I call the var.equals method with 2 parameters like this? And the ! negates the outcome?

File: rpc_shutdown/run.sh [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
+    for computer in $(bashio::config 'computers|keys'); do
+	    ALIAS=$(bashio::config "computers[${computer}].alias")
+	    ADDRESS=$(bashio::config "computers[${computer}].address")
+	    CREDENTIALS=$(bashio::config "computers[${computer}].credentials")
+	    DELAY=$(bashio::config "computers[${computer}].delay")
+	    MESSAGE=$(bashio::config "computers[${computer}].message")
+  
         # Not the correct alias
-        if [ "$ALIAS" != "$input" ]; then
+		if [! bashio::var.equals "$ALIAS" "$input" ]; then
```
Comment (pvizeli) [inline rpc_shutdown/run.sh]: Formatierung

File: rpc_shutdown/run.sh [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
             continue
         fi
       
-		# Check if delay is not empty 
-        if [ "$DELAY" = "" ]; then
+		# Check if delay is not empty
+		if bashio::var.is_empty "${DELAY}"; then
```
Comment (pvizeli) [inline rpc_shutdown/run.sh]: Formatierung

**Comment by stefanroelofs** [comment]
Found the syntax error. 
Also converted some tabs to spaces.

File: rpc_shutdown/run.sh (hunk 2 above)
Comment (stefanroelofs) [reply rpc_shutdown/run.sh]: You mean the tabs right? Fixed that.

**Comment by pvizeli** [comment]
Thanks

**Pull request merged**